example/policies/conduct.md

```diff
@@ -44,7 +44,7 @@ majorRevisions:
 
   a. *Corruption*
 
-      i. Employees are discouraged from accepting gifts from clients or partners. Briberies are prohibited for the benefit of any external or internal party. 
+      i. Employees are discouraged from accepting gifts from clients or partners. Bribery is prohibited for the benefit of any external or internal party. 
```

Comment:
I've never encountered the word `briberies`. Dictionaries do believe it exists, but while it is possible to use the plural form, I'm pretty sure this sentence/regulation works with the singular `bribery` to mean **any bribery is**.